e2e-tests/insurance/cypress/e2e/journeys/policy/complete-policy-as-multiple-contract-policy.spec.js

```diff
@@ -31,6 +31,7 @@ context('Insurance - Policy - Complete the entire section as a multiple contract
       cy.completeAndSubmitMultipleContractPolicyForm();
       cy.completeAndSubmitExportValueForm({ policyType });
       cy.completeAndSubmitNameOnPolicyForm({});
+      cy.completeAndSubmitPreCreditPeriodForm();
```

Comment:
should these go in a function since repeated a few times?

Reply:
I don't think so, for now at least. This is to get to the middle of a flow. There will be probably 5-7 other forms after this, for the policy section.

We have this everywhere - tricky one. Could have e.g `submitUpToXForm`, but then there would be lots of these commands for each stage of a section and if one of the forms is moved, it would be very troublesome to update/maintain. Unless there are any other ideas, I think it's better to have as we have now.